PxWeb/appsettings.json

```diff
@@ -21,8 +21,8 @@
     ],
     "DefaultLanguage": "en",
     "MaxDataCells": 10000,
-    "MaxCalls": 30,
-    "TimeWindow": 10,
+    "MaxCalls": 30,   // Note: Only for information - should match setting for IpRateLimiting
```

Comment:
We should remove them and map the configresponse from the settings in IpRateLimiting